external/vpx-flintstones/TheFlintstones(Williams1994)v1.26.ini

```diff
@@ -0,0 +1,22 @@
+[Player]
+BallAntiStretch = 1
+
+[TableOverride]
+ViewCabMode = 0
+ViewCabScaleX = 0.866888
+ViewCabScaleY = 1.100375
+ViewCabScaleZ = 1.099294
+ViewCabRotation = 90.000000
+ViewCabPlayerX = -48.500000
+ViewCabPlayerY = -0.000000
+ViewCabPlayerZ = -303.220062
+ViewCabLookAt = 5.000000
+ViewCabFOV = 52.879169
+ViewCabLayback = 48.000000
+
+[Standalone]
```

Comment:
these look like the DMD settings.

PinMAMEWindowX, PinMAMEWindowY should probably be in VPinballX.ini if they aren't already. We only want override settings in the per-table configs.

Reply:
Those are table positions to create a better POV. Not related to the DMD. I'll double check though as they may not be needed.

Reply:
It was just adjustments to the POV, it cleaned up black empty space at the top of the table. It's an adjustment users can make themselves though so fine to leave it out.

Reply:
No.. just wanted to make sure this wasn't something that should be applied everywhere. That should move into the main .ini.

Reply:
No. It should remain a per table setting.